### PR TITLE
Fix manual licensing

### PR DESCRIPTION
**Pull request by janeyx99**
It has come to my attention that some of our licenses are incorrect, so I attempted to rectify a few of them based on given recommendations for:
clog - BSD-3
eigen - MPL-2.0
ffnvcodec - LGPL-2.1
-> **hungarian - Permissive (free to use)**
irrlicht - The Irrlicht Engine License (zlib/libpng)
-> **pdcurses - Public Domain for core**
-> **sigslot - Public Domain**
test - BSD-3
Vulkan - Apache-2.0 or MIT
fb-only: more context is here https://fb.workplace.com/groups/osssupport/posts/26333256012962998/?comment_id=26333622989592967

This PR addressed the manual mismatches of licensing mentioned above (the two bolded, one is getting addressed in #128085, but as everything else is generated by pulling through other files, I did not address those. It is unclear what needs to be updated for the remaining to be accurate/if they're inaccurate today.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #128630

### CODE REVIEW

File: third_party/LICENSES_BUNDLED.txt [hunk 1]
```diff
@@ -78,6 +78,11 @@ License: BSD-3-Clause
 Files: third_party/kineto/libkineto/third_party/dynolog/third_party/DCGM/testing/python3/libs_3rdparty/colorama
   For details, see: third_party/kineto/libkineto/third_party/dynolog/third_party/DCGM/testing/python3/libs_3rdparty/colorama/LICENSE.txt
 
+Name: cpp-httplib
```
Comment (janeyx99) [inline third_party/LICENSES_BUNDLED.txt:81]: I'm not sure why this change got autogenerated from my changes...

**Review by malfet — APPROVED** [review]
LGTM, confirmed that by looking at https://github.com/microsoft/vcpkg/blob/fba75d09065fcc76a25dcf386b1d00d33f5175af/ports/hungarian/LICENSE.txt
and
https://github.com/microsoft/vcpkg/blob/fba75d09065fcc76a25dcf386b1d00d33f5175af/ports/pdcurses/LICENSE
